upcoming_changes/3175.new.rst

```diff
@@ -0,0 +1,2 @@
+Added in the `_onjumpclick` method to pointer widgets which allows the user to quickly go to some position
```

Comment:
As this is primiraly user facing, it would be to word as:
```suggestion
Add functionality to select navigation position using ``shift`` + click in the navigator.
```